test/mocks/services/mock_post_repository_service.dart

```diff
@@ -53,9 +53,13 @@ class MockPostRepositoryService extends Mock implements PostRepositoryService {
   }
 
   @override
-  Future<List<PostFirestore>> getNearPosts(GeoPoint? point, double? radius) {
+  Future<List<PostFirestore>> getNearPosts(
+    GeoPoint? point,
+    double? maxRadius, [
+    double? minRadius = 0.0,
```

Comment:
I think this should be tested in `post_repository_test.dart`.

Reply:
You're right. Done in 399ed8adda6ef7259ebf18ac451ed88e90355e73

Reply:
399ed8adda6ef7259ebf18ac451ed88e90355e73